.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+
+script:
+  - ./gradlew check
+  - ./gradlew test
+  - ./gradlew jacocoTestReport
```

Comment:
Do we need to run the tests in Travis to make swagger work?
If not can we remove these from all 3 projects -- they run in the pipeline and Travis doesn't always have the right dependencies for them to work